website/repairnator-site/README.md

```diff
@@ -17,6 +17,9 @@ $ npm install -g gulp bower
 $ npm install
 $ bower install
 
+# copy the config files (and edit them)
```

Comment:
why do you need multiple config files?

Reply:
So that you can specify different variables depending on your environment. 
Config handle different environments by loading the specific file and the default option file.

Reply:
why not writing the proper config file to deploy based on some objects contained in only one conf file? 

something like:

```json
{
  "constants": {
    "default": {
      "host": "http://defaulthost",
      "port": "4444"
    },
    "dev": {
      "host": "http:localhost",
      "port": "88"
    },
    "prod": {
      "host": "http://fancyurl",
      "port": "80"
    }
}
```

then you have only one file to rely one for all config.

Reply:
It is also an option. But then I would not be using [config](https://www.npmjs.com/package/config) anymore. 
But it is no problem I can also create a small config loader to handle this.

Reply:
hm indeed I didn't check that you were using this lib. Your call then: it's really a detail here. I can merge the PR like this.

Reply:
I think both solutions are good so your can merge like that.